issuing/client_base.go

```diff
@@ -0,0 +1,18 @@
+package issuing
+
+import (
+	"github.com/checkout/checkout-sdk-go/client"
+	"github.com/checkout/checkout-sdk-go/configuration"
+)
+
+type Client struct {
+	configuration *configuration.Configuration
+	apiClient     client.HttpClient
+}
+
+func NewClient(configuration *configuration.Configuration, apiClient client.HttpClient) *Client {
```

Comment:
I don't know about this. I think we should stick with one client for all issuing operations.

Reply:
👍🏽